### ISSUE TITLE
Map overlay graphics for review

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#903]
A few options for the homepage. 

There are some considerations to work out, ie: 
- what will happen if you click the cases tab - should the map activate automatically? 
- Could this area be a scrolling slideshow, like on .net? 
- What could the visual be on tabs where the map doesn't make sense, or do we just leave a blank map or this visual there? 

As a beginning for visuals to consider, I was thinking that after activating the map, a small red info icon in the bottom right corner of the map could bring the overlay back up if people wanted to see it again. And, when re-sizing screens, in option A the content would move towards the centre of the screen, and options B and C the large text would disappear and the popup and anyone can join text would move to centre. 

Feedback welcome!

Info icon on map:
![all - map info icon](https://user-images.githubusercontent.com/13560530/51054108-89083080-1590-11e9-9316-3bbddb6da8c0.jpg)

Option A:
![all - map overlay a](https://user-images.githubusercontent.com/13560530/51054112-932a2f00-1590-11e9-8659-6a069a859485.jpg)

Option B:
![all - map overlay - b](https://user-images.githubusercontent.com/13560530/51054123-9b826a00-1590-11e9-8b82-83f8f07f2a48.jpg)

Option C:
![all - map overlay - c](https://user-images.githubusercontent.com/13560530/51054132-a3daa500-1590-11e9-95dc-30e42673eba1.jpg)

**Comment by scottofletcher** [comment]
Not a fan of B or C - there's too much text to take in. Ideally, the 'tag line' should be front-and-centre and needs to be updated to "A collaborative effort to document public participation and democratic innovation around the world". Maybe the version on the map doesn't need the "around the world" part though? The map kind of makes that obvious. 

I'm also not a huge fan of having "Public participation occurs when citizens collaborate to influence civic life" for a couple reasons: 
1 - 'citizenship' is more of a Western/Global North concept. Public participation also occurs in non-governmental settings (including organizations) where citizenship may be out of reach for many/most or it may not exist in the same way we conceive of it (ie. as a 'license' - often but not always received at birth) to participate and benefit from the actions of other community members as an equal)
2 - influencing civic life is not a requisite of public participation. This is more of a poli-sci definition (ie. "policy/measurable political change or it didn't happen"). we have cases of informal participation that had their _direct_ influence(s) on personal/private life (ie. outside of the public/civic sphere). 
3 - formatting to look like a pin on the map which makes me question the utility of the other pins (ie. are they just for show? are they cases?)

As well, I would avoid defining or narrowing the definition of public participation. even though Democratic Innovation, Participatory Democracy, Civic Engagement, and Participatory Governance are all appreciably broad, we've learned that our readers take things literally - as in the case of the heading "Deliberation, Decisions, and Public Interaction" (which was similarly meant as a 'catch all' for ever conceivable type of participation but which clearly failed).

I would be in favour of A with some changes: 
- replace "Public participation occurs..." with "A collaborative effort to document public participation and democratic innovation" (either with or without the prefix "Participedia is" or the ending "around the world")
- add a comma after 'edit' in the sentence "Use Participedia to publish, edit and share..."
- change "Click to activate map" to "Click anywhere to activate map" (which is what I assume you meant and not "click the little map pin to the right of this message" or "click the red pin in the centre" but I'm not sure)
- change the background colour to the one used in B & C

Random suggestion: maybe the Login button at the top right should be red like "Join Now"?

**Comment by jesicarson** [comment]
@scottofletcher what do you think about removing the tag line from below the map area - placing it on the map overlay? so new users would see it - BIG, and you could always click the info icon to bring it back if you wanted to. This would be rather than having the tag line twice - once on the overlay and then consistently below it. What do you think?
I also agree that the map pin thing could be confusing and that people take things literally. Do we want to avoid placing our own definition on public participation at all? i just wonder if people get what "public participation" is right away, and was thinking if we define it or give an example it might help. I think at least there should be an FAQ question under the help menu that answers "What is public participation?"

**Comment by scottofletcher** [comment]
@jesicarson RE tag line placement: sorry that's what I meant! Here are the elements I'd combine: 
- Dark background of B
- Tag line from below map area moved to map overlay (replaces large definition of public participation)
- Join now button + "anyone can join...use PPedia to publish..."
> change that last part to "Anyone can publish, edit, and share Participedia entries"

RE definition: avoid for now. we're actually working on a glossary of terms like 'public participation', 'deliberation', 'democratic education', (etc.) but we need all of our partners to weigh in first. right now there are just too many definitions and too much disagreement over those definitions to put anything out there. for now, let's adapt the list of 'general types of methods' and/or 'general types of tools' to give examples of what participation looks like. For example: 
Public participation occurs when:
- two or more stakeholders work together to address a common problem
- communities drive, implement, and effect change)
- people take part in structured processes that involve deliberation and/or dialogue as a central aspect)
- citizens exert direct authority over decisions
- public bodies and services are monitored, evaluated, and officials and authorities are held to account
- teachers and instructors use experiential and immersive education 
- access to resources, rights and political representation are secured without using formal governmental channels
- organizations and bodies adopt systems of internal communication, representation, or decision-making
ETC

Does that work? When would you need a full list by?

**Comment by jesicarson** [comment]
Ok this is good. Thanks! 
Regarding a definition, I think that list is great for the FAQ section, and it can always be updated.

**Comment by scottofletcher** [comment]
those were just the first few examples, I'd like to revise the list before it gets put up

**Comment by jesicarson** [comment]
Sounds good. You can post the list whenever you're ready in #904 and the copy deck.

**Comment by plscully** [comment]
@jesicarson @scottofletcher I think we're getting close on this. I prefer removing the tag line from below the map area and placing it on the map overlay. I created this doc in the Copy Deck folder with a summary of possible text for the overlay & text in lower left corner of map.  That should be a better place to word craft than here.  We can also wait a while to work on the best way to describe/define public participation, and to decide where that text belongs. https://docs.google.com/document/d/1Nmdagogb04JXzYkCFdVnd3ckpzWevl3TbjPJA6G6kzc/edit?usp=sharing

**Comment by jesicarson** [comment]
@scottofletcher @plscully 
Updated designs E & D for review:

Option D
![all - map overlay - d](https://user-images.githubusercontent.com/13560530/53269734-7ceab500-369e-11e9-8792-c9c5a0e884c0.jpg)

Option E
![all - map overlay - e](https://user-images.githubusercontent.com/13560530/53269742-8411c300-369e-11e9-9d69-983c65141925.jpg)

**Comment by plscully** [comment]
I like both options and don't have a strong opinion about which one is preferable. It's just the font size that differentiates them, right? 
@jesicarson Can you please run this by a couple of people with good eyes for design and see what they prefer? Thanks!

**Comment by jesicarson** [comment]
Thanks Pat will do!

**Comment by scottofletcher** [comment]
I like the second version! Can the Join Now button be bigger, though? And can it be centred over the "Anyone can..." section?

**Comment by plscully** [comment]
@scottofletcher I think the Join Now button is positioned so that its left-hand margin aligns with the overlay and the cc. That is probably the cleanest way to present it. I don't have an opinion about the size of the button.

**Comment by scottofletcher** [comment]
@plscully yes, I see that it's aligned to the CC. I was just speaking from personal opinion that I think it would look better if it was 1) larger, and 2) either aligned with "Anyone can" or was centred over that paragraph. whatever the design-folks think is best though.

**Comment by jesicarson** [comment]
F
![all - map overlay - f](https://user-images.githubusercontent.com/13560530/53512665-2f908e00-3a78-11e9-9c73-5bc4a1e37643.jpg)

G
![map overlay g](https://user-images.githubusercontent.com/13560530/53512677-38815f80-3a78-11e9-8e29-b51d1f0711ea.jpg)

H
![map overlay h](https://user-images.githubusercontent.com/13560530/53512687-3e774080-3a78-11e9-85c6-358be4085de3.jpg)

**Comment by scottofletcher** [comment]
Oh my gosh I love it! I can't decide which I like better. F is good but it's missing the map key (and adding it might make things too busy). G is probably my favourite, but could we use the new tag line?

**Comment by plscully** [comment]
I like Option H the best, but with Option G's styling of  the longer, thinner "join now" with the pins below it.

**Comment by plscully** [comment]
Yes, so I agree with Scott that G is best, but with the new tag line

**Comment by scottofletcher** [comment]
tag line's good, yeah? mashup of a bunch of press release intros Jesi and I drafted!

**Comment by jesicarson** [comment]
I think its getting there! Not 100% sold on the word "written" because not all of the contribution is about writing (leaves out contributions of media and data), but does rhyme, which is cute :P 

Here's the latest (OPTION I) with your input. I will be running this by Amber Thurs and will let you know if she has feedback. 

![map overlay i](https://user-images.githubusercontent.com/13560530/53515174-3cb07b80-3a7e-11e9-9101-5a77dcdc42d9.jpg)

Here's the homepage showing google map once the overlay is clicked (displays info icon in bottom left corner, which if clicked would bring back the overlay):

![google map w info icon bottom left](https://user-images.githubusercontent.com/13560530/53515222-5651c300-3a7e-11e9-990c-66118e2bb601.jpg)

**Comment by plscully** [comment]
We could substitute either "created" or "produced" for "written".  Created seems better to me; it rhymes a bit and it plays off our "creative commons" stance

**Comment by scottofletcher** [comment]
'community-driven, collaboratively created' just doesn't sound right... I think it's one too many 'c's! what about 'generated'? or 'authored', 'recorded'? otherwise, I like 'produced'
I personally don't think 'written' implies just text, but maybe that's just me. I think people get that, by 'written', we mean compiled/created. the enhanced visuals of the site imply that images/multi-media are a part of 'documenting'.

**Comment by jesicarson** [comment]
@ascott latest design files [here](https://drive.google.com/open?id=1jNEWeatdj-qFNxgS9X4Jzi-drPWoBwDA)

**Comment by jesicarson** [comment]
Latest mockups (OPTION J)

Homepage Default (overlay)
![Map Overlay - Home Default](https://user-images.githubusercontent.com/13560530/57332754-408cf980-70d0-11e9-8d0d-4d1b90996b67.jpg)

Homepage with Map Activated (overlay can be reactivated by clicking the info icon)
![Map Overlay - Map Activated](https://user-images.githubusercontent.com/13560530/57332755-41259000-70d0-11e9-8086-7977e498c9da.jpg)

Mobile design in progress
![Screen Shot 2019-05-07 at 1 56 32 PM](https://user-images.githubusercontent.com/13560530/57332690-1d624a00-70d0-11e9-977c-acb39efddf63.png)

**Comment by plscully** [comment]
Looking good! Very nice!

**Comment by scottofletcher** [comment]
@jesicarson @ascott The tagline needs to be changed at the bottom of the page as well

**Comment by jesicarson** [comment]
@scottofletcher @plscully FYI: Further review by Amber of the tagline has led us to defer back to the original tagline: "A global community sharing knowledge and stories about public participation" because it is shorter visually as well as easier to digest informationally, at-a-glance. I will also be updating the visual design of the overlay following the review by Amber today.

**Comment by jesicarson** [comment]
Post review, updated draft "Option F" from an earlier iteration, now OPTION K:

OPTION K - Homepage default
![Map Overlay F updated-Homepage Default](https://user-images.githubusercontent.com/13560530/57406178-076a8d00-7195-11e9-851b-e917327a2155.jpg)

OPTION K - Map activated with legend (join now button persists until logged in)
![Map Overlay F updated-Activated Map not logged in](https://user-images.githubusercontent.com/13560530/57406194-0df90480-7195-11e9-8784-7a75a949e845.jpg)
